tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_custom_StyledTextContent.java

```diff
@@ -87,16 +87,14 @@ public void setText(String text) {
 
 @Before
 public void setUp() {
-	if (SwtTestUtil.isBidi()) XINSET = 2;
-	else XINSET = 0;
 	shell = new Shell();
 	styledText = new StyledText(shell, SWT.NULL);
 	styledText.setContent(content);
 }
 
 @Test
 public void test_getCharCount() {
-	assertEquals(":a:", 0, styledText.getCharCount());
+	assertEquals(":a:", INITIAL_TEXT.length(), styledText.getCharCount());
```

Comment:
While at it, please remove these ":a:" messages. They are totally useless and what's worse cause confusion when converting tests to Java 5 where assert* methods moved message as last parameter.